lib/companies_web/router.ex

```diff
@@ -29,6 +29,10 @@ defmodule CompaniesWeb.Router do
   scope "/", CompaniesWeb do
     pipe_through [:browser, :set_locale]
 
+    if Mix.env() == :dev do
```

Comment:
Not in love with dev-specific code that is in the main code. But its whatever. Curious question: Will the compiler ditch this whole block when compiling with MIX_ENV=prod since it would evaluate to false?

Reply:
Me either. I'm otherwise sure how to handle this one specific dev route.

I do not believe the compiler will remove this call.

Reply:
Any suggestions?